freva-rest/entrypoint.sh

```diff
@@ -60,7 +60,7 @@ wait_for_mongo() {
 
 reset_mongo_user() {
     log_info "Resetting MongoDB user..."
-    mongosh admin --quiet --eval "
+    bash mongosh admin --quiet --eval "
```

Comment:
What are those `bash` statements for?

Reply:
when I switched to alpine, an action specific issue raised which is not reproducible on my machine. I meant it works here but not on action. So its simply a tryout to fix. But I’m on it to find a solution …